index.html

```diff
@@ -23,9 +24,15 @@
     <meta property="og:title" content="Virtual Assistant | Assistant Virtuel" />
     <meta property="og:description" content="Virtual Assistant | Assistant Virtuel" />
 
+    <!-- dcterms metadata -->
+    <meta name="dcterms.creator" content="ESDC_GC"/>
```

Comment:
I imagine we should follow what Alpha did: have a locale for the content.

en: Employment Social Development Canada.

fr: Emploi et Développement social Canada

Reply:
I'm not sure how we'd add a locale for the index page since it's not a Vue file...

Reply:
Hummm good question.

I found this link that could help: https://www.epiloge.com/how-to-add-dynamic-meta-tags-to-your-vuejs-app-for-google-seo-0fa058

Reply:
vue-meta doesn't have vue 3 support and even though the latest alpha version does, it doesn't seem to have any documentation. I modified the meta tag directly with JS instead for now.